### PR TITLE
component badges

### PR DESCRIPTION
**Pull request by SKeerthi02**
Features 
1. Badges on Avatars.
2. Badges on Icons.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -29,3 +29,79 @@ body{
...
+/* badge icon CSS*/
+
+.badge-icon{
+    display: flex;
+}
+.badge-icon-primary{
+    color: var(--primary-color);
+}
+.cart-quantity{
+    font-size: 20px;
```
Comment (Aditi0505) [inline styles.css]: Can use rem here too for consistency.

File: styles.css [hunk 2]
```diff
@@ -29,3 +29,79 @@ body{
...
+}
+.badge-busy{
+    background-color: var(--busy);
+}
+.badge-away{
+    background-color: var(--away);
+}
+
+.badge-circle-away{
+    font-size: 16px;
```
Comment (Aditi0505) [inline styles.css]: Can use rem here too for consistency.

File: styles.css [hunk 3]
```diff
@@ -29,3 +29,79 @@ body{
...
+    width: 75px;
+    height: 75px;
+}
+.badge-avatars {
+    display: flex;
+    flex-wrap: wrap;
+}
+.badge-circle-time{
+    width: 24px;
+    height: 16px;
```
Comment (Aditi0505) [inline styles.css]: Can use rem here too for consistency.

File: styles.css [hunk 4]
```diff
@@ -29,3 +29,79 @@ body{
...
     cursor: pointer;
 }
+
+/* badges */
+.badge-avatar{
+    display: flex;
+}
+.badge-circle{
+    width: 16px;
+    height: 16px;
```
Comment (Aditi0505) [inline styles.css]: Can use rem here too for consistency.

File: styles.css [hunk 5]
```diff
@@ -29,3 +29,79 @@ body{
...
+    border-radius: 50%;
+    position: relative;
+    bottom: -3rem;
+    left: -1rem;
+}
+
+.online-avatar{
+    vertical-align: middle;
+    width: 75px;
+    height: 75px;
```
Comment (Aditi0505) [inline styles.css]: Can use rem here too for consistency.

File: styles.css [hunk 6]
```diff
@@ -29,3 +29,79 @@ body{
...
+    display: flex;
+}
+.badge-icon-primary{
+    color: var(--primary-color);
+}
+.cart-quantity{
+    font-size: 20px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (Aditi0505) [inline styles.css]: I observe this piece of code is repeating. You can create a class for this.

**Review by Aditi0505 — APPROVED** [review]
Left few suggestions. Rest looks good to me.
- You can use rem units for values above 10px. 
- You can use px units for small values less than 10px.


Also, you can provide description and screenshots for better visualization.

File: styles.css (hunk 6 above)
Comment (SKeerthi02) [reply styles.css]: Thank you, Changed it.

File: styles.css (hunk 5 above)
Comment (SKeerthi02) [reply styles.css]: addressed and changed at all positions.

**Pull request merged**